### ISSUE TITLE
[Compatibility] OptiFine no longer working with Controllable/Framework in 1.20.1

### ISSUE THREAD

**Issue by bishopsknfetrck** [sp614x/optifine#7451]
## Description of Issue
Minecraft 1.20.1 is crashing when I try to use OptiFine with the Controllable mod, which as of 1.20.1, now requires a separate library called "Framework". I am running Controllable 0.20.2, as 0.20.3 requires Forge 47.0.4, and I am running Framework 0.6.5

## Steps to Reproduce
1. Load Minecraft with just Controllable and OptiFine installed, it gives me a crash menu requesting that I install Framework
2. Install Framework, and reboot Minecraft, which totally crashes back to the launcher, and gives me Exit Code 1.
3. Use the Thanos Method to isolate the problem mod, and since we already know that Controllable requires Framework to run, I remove OptiFine and boot the game.
4. It loads smoothly. Seeing as Minecraft tries to load when it's just Controllable, the issue lies in Framework

## OptiFine Version
preview_OptiFine_1.20.1_HD_U_I5_pre6

## Installation Method
Dropped into the "mods" folder as a JAR file

## Fabric/Forge Version
Forge 47.0.4

## Other Installed Mods
Cloth Config-11.0.99-forge
Controllable-forge-1.20.1-0.20.2
coordinatesdisplay-2.3.4
FallingTree-120.1-4.1.1
Framework-forge-1.20.1-0.6.5

## Log Files/Crash Reports
![image](https://github.com/sp614x/optifine/assets/132172222/402a720e-9cc1-4a06-bc13-09a19cdb503e)

## Prior Testing
Prior testing was documented in the steps to reproduce.

## Additional Information
I brought this up in the Discord and asked if I should make a GitHub report about it, and was told yes, so that's what I've done.

**Comment by YkVox** [comment]
optifine's forge version crashes instantly, regardless of the other mods...
i hope it will get fixed along further preview versions

**Comment by imLeonam** [comment]
I'm having the same problem, will it be fixed?

**Comment by Coder-Channel** [comment]
Same issue here. OptiFine HD U I5 works on Forge 47.0.35, but none of the preview versions work on the specified Forge version.

**Comment by Coder-Channel** [comment]
I think I found the culprit. I noticed that when I reset the optionsof file for unrelated reasons, the error went away. By changing the options a bunch until the game broke, I realised that antialiasing being turned on crashed the game. The error in question:
![Скриншот 31-08-2023 011438](https://github.com/sp614x/optifine/assets/33266814/ccfc7efd-ddce-4ca5-9956-34f89fb1f3dc)

So if anyone is experiencing this issue **go into the optionsof.txt file in your .minecraft folder and set ofAaLevel to 0.**

I presume this *might* be an issue caused by AMD drivers (mine are up to date).
